src/els_compiler_diagnostics.erl

```diff
@@ -82,13 +82,18 @@ compile(Uri) ->
 -spec parse(uri()) -> [diagnostic()].
 parse(Uri) ->
   FileName = binary_to_list(els_uri:path(Uri)),
-  {ok, Epp} = epp:open([ {name, FileName}
-                       , {includes, els_config:get(include_paths)}
-                       ]),
-  Res = [diagnostic(range(Line), Module, Desc, ?DIAGNOSTIC_ERROR)
-         || {error, {Line, Module, Desc}} <- epp:parse_file(Epp)],
-  epp:close(Epp),
-  Res.
+  case epp:open([ {name, FileName}
+                , {includes, els_config:get(include_paths)}
+                ]) of
+      {ok, Epp} ->
+          Res = [diagnostic(range(Line), Module, Desc, ?DIAGNOSTIC_ERROR)
+                 || {error, {Line, Module, Desc}} <- epp:parse_file(Epp)],
+          epp:close(Epp),
+          Res;
+      _ ->
```

Comment:
I think we could (as you suggest) show an error to the user in this case. Something like:

    "Could not parse the included file ([ERROR HERE]).

Reply:
We can do one (or both) of

1. Generate a diagnostic
2. Send a `showMessage`

We might not be able to attach the diagnostic to a file in the IDE, depending on what precisely the problem is.  But I think they should show up somewhere anyway, I am pretty sure they do in vscode.

Reply:
In case of issues within the hrl file, we already override the location of the original diagnostic message and display the message in the inclusion line:

https://github.com/erlang-ls/erlang_ls/blob/master/src/els_compiler_diagnostics.erl#L126

We could do the same with this. Thoughts?

Reply:
All we have here is the Uri we are trying to parse, and we know that it failed, because the file does not exist, we are not searching in the right location, etc.

This should not actually happen, but if it does it indicates a misconfiguration of the project.  So I think a `window/ShowMessage` of type `ERROR` should be sent.